drafttopic/utilities/tests/test_fetch_wikiprojects.py

```diff
@@ -11,7 +12,7 @@
 
 
 def test_wiki_projects_from_table():
-    table = fetch_section_text('performing_arts.test')
+    table = fetch_section_text('performing_arts.test', logger)
```

Comment:
Generally, you don't want to pass a logger around.  It's OK to let each .py file have it's own logger.